crates/tokio_helper/src/lib.rs

```diff
@@ -0,0 +1,79 @@
+use tokio::runtime::Runtime;
+
+lazy_static::lazy_static! {
+    pub static ref TOKIO: Runtime = {
+        new_runtime()
+    };
+}
```

Comment:
```suggestion
use once_cell::sync::Lazy;

pub static TOKIO: Lazy<Runtime> = Lazy::new(new_runtime);
```

once_cell is the new lazy_static! : )